### PR TITLE
Feature: Schema title bar

### LINKED ISSUE #2202: Schema: Title Bar Revamp
# Intro 
We will make the following title bars in Schema match the new material styling being followed in the new Content Title Bar.

1. Edit Model
2. All Models
3. Search Results

image
image
image

In addition to this, we will update: 

1. The option labelled "Copy ZUID" in the more dropdown menu of the Titlebar to read as "Copy Model ZUID". 
2. The option labelled "Copy ZUID" in the more dropdown menu of a field to read as "Copy Field ZUID"

This will provide more clarity to a user about what ZUID they are copying. 

# Figma 
https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/App-Navigation?type=design&node-id=917%3A237213&mode=design&t=AudchdNfHGSJJxQl-1

### LINKED ISSUE #2226: Content: Dashboard Title Bar
The title bar for the Content Dashboard has been updated to match the new Title bar styling being used in the global Title Bar update. 

# Figma
https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/App-Navigation?type=design&node-id=1104%3A254994&mode=design&t=EG7uR9iUYa1VP3Rf-1

### LINKED ISSUE #2247: Replace All Models Icon with Database Search Icon to make it more intuitive
We will update the icon of All Models in the Sidebar and in the Breadcrumb to be database-search instead of the currently used Schema Icon.

image

### PR DESCRIPTION
**Pull request by theofficialnar**
Closes #2202 
Closes #2226 
Closes #2247 

**Dependent on https://github.com/zesty-io/material/pull/80 for the new icon** 

#### Preview
all models
search
model view
field select
field input

### CODE REVIEW

File: src/apps/schema/src/app/components/AddFieldModal/views/FieldForm.tsx [hunk 1]
```diff
@@ -464,17 +465,13 @@ export const FieldForm = ({
   return (
     <>
       <DialogTitle
+        component="div"
         sx={{
-          padding: 0,
+          borderBottom: `2px solid ${theme.palette.border}`,
```
[the remark this answers is not in the document]
Comment (theofficialnar) [reply src/apps/schema/src/app/components/AddFieldModal/views/FieldForm.tsx]: Updated, changed to `borderColor: "border"` instead

**Review by zcolah — CHANGES_REQUESTED** [review]
1. [Visual QA Feedback on Figma](https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/Zesty-3.0-Revamp?type=design&node-id=1937%3A217375&mode=design&t=myTMEkSKQfqXaqm5-1)

2. [Make Info Tab Updates in Figma](https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/Zesty-3.0-Revamp?type=design&node-id=1958%3A264624&mode=design&t=QXBqnuJ6L18Q7Y3L-1): There is a poor contrast on the Info Tab page due to the change in the background color to Grey.50. I had not anticipated this and hence need to request 4 changes to be made to the content items in the Info Tab. Apologies for the same. 

3. Update Icon of All Models in the Sidebar and in the Breadcrumbs to be database-search (sent the icon SVG on Slack). Please also ensure this PR is connected to this [issue](https://github.com/zesty-io/manager-ui/issues/2247).  See screenshot below

image

**Review by zcolah — CHANGES_REQUESTED** [review]
**Info Tab Update**
There is a poor contrast on the Info Tab page due to the change in the background color to Grey.50. I had not anticipated this and hence need to request 4 changes to be made to the content items in the Info Tab. 

See Figma design here: https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/Zesty-3.0-Revamp?type=design&node-id=1958%3A264624&mode=design&t=QXBqnuJ6L18Q7Y3L-1

Apologies for the same.

**Review by zcolah — CHANGES_REQUESTED** [review]
Please update the icon of All Models in the Sidebar and in the Breadcrumb to be database-search. I have sent the icon SVG via Slack. 

image

**Review by zcolah — CHANGES_REQUESTED** [review]
NA

**Review by zcolah — CHANGES_REQUESTED** [review]
Overall Great Job @theofficialnar, please make the following changes to the designs.

1. [Visual QA Feedback on Figma](https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/Zesty-3.0-Revamp?type=design&node-id=1937%3A217375&mode=design&t=myTMEkSKQfqXaqm5-1)

2. [Make Info Tab Updates in Figma](https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/Zesty-3.0-Revamp?type=design&node-id=1958%3A264624&mode=design&t=QXBqnuJ6L18Q7Y3L-1): There is a poor contrast on the Info Tab page due to the change in the background color to Grey.50. I had not anticipated this and hence need to request 4 changes to be made to the content items in the Info Tab. Apologies for the same. 

3. [Update Icon of All Models in the Sidebar and in the Breadcrumbs](https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/Zesty-3.0-Revamp?type=design&node-id=917%3A237213&mode=design&t=QXBqnuJ6L18Q7Y3L-1) to be database-search (sent the icon SVG on Slack). Please also ensure this PR is connected to this [issue](https://github.com/zesty-io/manager-ui/issues/2247).  See screenshot below

image

4. Update API Tab Padding Issues. Currently the API Tab spacing looks to be off. So we need to make two changes specifically as listed below. You can also refer to this [Figma file](https://www.figma.com/file/zOI7oSH3fG1XDmmPzsP6PX/Zesty-3.0-Revamp?type=design&node-id=1960%3A265835&mode=design&t=QXBqnuJ6L18Q7Y3L-1) to see the spacing rules.

  a. Make the top padding on this div from 16px to 0px. 
  CleanShot 2023-09-06 at 19 43 56@2x

  b. Make the bottom padding on this div from 20px to 16px
  CleanShot 2023-09-06 at 19 45 30@2x

**Review by zcolah — CHANGES_REQUESTED** [review]
1. Update the info icon size for the description field in the right sidebar to be 12px by 12px. Note: This is a bug not originally in scope. 

2. Make the color of the title in each field list item to be typography.primary. 

CleanShot 2023-09-08 at 04 18 22@2x

3. Please rename Copy ZUID to Copy Model ZUID

CleanShot 2023-09-08 at 04 19 36@2x

4. Please rename copy zuid here to "Copy Field ZUID"
CleanShot 2023-09-08 at 04 20 00@2x

**Pull request merged**